python/xorbits/_mars/deploy/oscar/supervisor.py

```diff
@@ -52,6 +52,21 @@ def parse_args(self, parser, argv, environ=None):
 
         args.supervisors = f"{args.supervisors},{args.endpoint}".strip(",")
 
+        oscar_config = self.config.get("oscar")
+        external_addr_scheme = None
+        if oscar_config is not None:
+            numa_addr_scheme = oscar_config.get("numa").get("external_addr_scheme")
+            gpu_addr_scheme = oscar_config.get("gpu").get("external_addr_scheme")
+            external_addr_scheme = numa_addr_scheme or gpu_addr_scheme
```

Comment:
Correct me if I'm wrong: we shouldn't decide whether the supervisor is prefixed with ``ucx://`` based on the schema of the GPU (``gpu_addr_scheme``), since ``gpu_addr_scheme`` only decides subprocesses on GPU to do the communications. So just ``external_addr_scheme = numa_addr_scheme`` for this line.

Reply:
Maybe you are right. I did not test on GPUs. Later I will test it on GPUs.

Reply:
I think we can first solve the CPU problem as GPU communication is much more complicated. There are many ways for GPU communications (nvlink, ib, cuda-ipc, etc). What's more, multi-GPUs are not well supported in this communication model. We can test GPUs later.  Just fix the CPU problem with this PR.

Reply:
> 

Sure. Thanks. This is fine for now.